src/lib/Datasource.js

```diff
@@ -0,0 +1,6 @@
+class Datasource {
```

Comment:
Please, consider to add 'use strict' line at the start of every file.

Reply:
done